Riot/AppDelegate.m

```diff
@@ -1210,8 +1226,14 @@ - (void)launchBackgroundSync
                 
             } failure:^(NSError *error) {
                 
-                NSLog(@"[AppDelegate][Push] launchBackgroundSync: the background sync fails");
-                
+                NSLog(@"[AppDelegate][Push] launchBackgroundSync: the background sync failed. Error: %@ (%@). incomingPushEventIdsCopy: %@ - self.incomingPushEventIds: %@", error.domain, @(error.code), incomingPushEventIdsCopy, incomingPushEventIds);
+
+                // Trigger limited local notifications when the sync with HS fails
+                [self handleLimitedLocalNotifications:account.mxSession events:incomingPushEventIdsCopy];
```

Comment:
Are we sure here that no incoming push event has been handled? (I think so, but...)

Reply:
Nope :/
Specially, if MXSession needs to do several syncs to complete the catch-up. 1st syncs succeeded but only last one failed.

Reply:
Fix by https://github.com/vector-im/riot-ios/pull/1714/commits/0cdcff95fce7d17354b3adea6d7bc5bd05eafc97